tasklist/importer-860/src/main/java/io/camunda/tasklist/zeebeimport/v860/processors/es/UserTaskZeebeRecordProcessorElasticSearch.java

```diff
@@ -130,4 +135,72 @@ private UpdateRequest getVariableQuery(final TaskVariableEntity variable)
           e);
     }
   }
+
+  private UserTaskListViewEntity createTaskListViewInput(final TaskEntity taskEntity) {
+    final UserTaskListViewEntity userTaskListViewEntity = new UserTaskListViewEntity();
+    Optional.ofNullable(taskEntity.getFlowNodeInstanceId())
+        .ifPresent(userTaskListViewEntity::setId); // The ID is necessary for the join
+    Optional.ofNullable(taskEntity.getFlowNodeInstanceId())
+        .ifPresent(userTaskListViewEntity::setFlowNodeInstanceId);
+    Optional.ofNullable(taskEntity.getProcessInstanceId())
+        .ifPresent(userTaskListViewEntity::setProcessInstanceId);
+    Optional.ofNullable(taskEntity.getFlowNodeBpmnId())
+        .ifPresent(userTaskListViewEntity::setTaskId);
+    Optional.ofNullable(taskEntity.getFlowNodeBpmnId())
+        .ifPresent(userTaskListViewEntity::setFlowNodeBpmnId);
+    Optional.of(taskEntity.getKey()).ifPresent(userTaskListViewEntity::setKey);
+    Optional.of(taskEntity.getPartitionId()).ifPresent(userTaskListViewEntity::setPartitionId);
+    Optional.ofNullable(taskEntity.getCompletionTime())
+        .map(Object::toString)
+        .ifPresent(userTaskListViewEntity::setCompletionTime);
+    Optional.ofNullable(taskEntity.getAssignee()).ifPresent(userTaskListViewEntity::setAssignee);
+    Optional.ofNullable(taskEntity.getCreationTime())
+        .map(Object::toString)
+        .ifPresent(userTaskListViewEntity::setCreationTime);
+    Optional.ofNullable(taskEntity.getProcessDefinitionVersion())
+        .ifPresent(userTaskListViewEntity::setProcessDefinitionVersion);
+    Optional.ofNullable(taskEntity.getPriority()).ifPresent(userTaskListViewEntity::setPriority);
+    Optional.ofNullable(taskEntity.getCandidateGroups())
+        .ifPresent(userTaskListViewEntity::setCandidateGroups);
+    Optional.ofNullable(taskEntity.getCandidateUsers())
+        .ifPresent(userTaskListViewEntity::setCandidateUsers);
+    Optional.ofNullable(taskEntity.getBpmnProcessId())
+        .ifPresent(userTaskListViewEntity::setBpmnProcessId);
+    Optional.ofNullable(taskEntity.getProcessDefinitionId())
+        .ifPresent(userTaskListViewEntity::setProcessDefinitionId);
+    Optional.ofNullable(taskEntity.getTenantId()).ifPresent(userTaskListViewEntity::setTenantId);
+    Optional.ofNullable(taskEntity.getExternalFormReference())
+        .ifPresent(userTaskListViewEntity::setExternalFormReference);
+    Optional.ofNullable(taskEntity.getCustomHeaders())
+        .ifPresent(userTaskListViewEntity::setCustomHeaders);
+    Optional.ofNullable(taskEntity.getFormKey()).ifPresent(userTaskListViewEntity::setFormKey);
+    Optional.ofNullable(taskEntity.getState()).ifPresent(userTaskListViewEntity::setState);
```

Comment:
Not need to check Optional.ofNullable(field).ifPresent(setTheField)
all the fields in `new UserTaskListViewEntity()` are null after initialization, so a `null` value will not override an existing value in UserTaskListViewEntity
I think it would be good to have a mapper constructor `UserTaskListViewEntity(TaskEntity)`